graphstorm-processing/docker/0.2.1/sagemaker/Dockerfile.cpu

```diff
@@ -0,0 +1,45 @@
+# syntax=docker/dockerfile:experimental
```

Comment:
Can you add sagemaker docker support In another PR?

Reply:
You mean move this entire file to another PR?